sdcm/cluster_docker.py

```diff
@@ -52,12 +52,18 @@ def node_container_run_args(self, seed_ip):
         volumes = {
             '/var/run/docker.sock': {"bind": '/var/run/docker.sock', "mode": "rw"},
         }
+
+        # TODO: add parsing of cpuset option from append_scylla_args parameter
+        scylla_args = self.parent_cluster.params.get('append_scylla_args')
```

Comment:
We are now doing it for all nodes, including loader nodes, so we might want to limit it per node type, and do it just for the db one for now, and think how it would be better to control each one on its own